Pages/Healthcare.cshtml

```diff
@@ -43,23 +44,21 @@
         <tr>
             <th>Healthcare Center<br />
             <th>Neighborhood</th>
-            <th>Services</th>
-            <th>Street Name</th>
+            <th>Services Offered</th>
+            <th>Street</th>
             <th>City</th>
             <th>Zip Code</th>
-
+            
         </tr>
         @{
             foreach (QuickType1.Healthcare healthcare in healthcares)
             {
                 <tr>
 
-                   
-
 
                     <td>@healthcare.HealthCenter</td>
                     <td>@healthcare.Neighborhood</td>
-                    <td>@healthcare.ServicesProvided</td>
+                    <td style="text-align:left">@healthcare.ServicesProvided</td>
```

Comment:
Aligned the description text to left

Reply:
Good idea, but go one step further.  Move to external/linked CSS file.